Update readme carbon (#73)
bow-swift/nef: README.md

struct Person {} // This will be present in the Markdown file
```
</details>

&nbsp;

### 🌁 Exporting Carbon code snippets

> Xcode Playgrounds are a great place where prototyping and trying new concepts. Many times we want to share some Swift snippets, for this `Carbon` is a cool tool and `nef` lovely integrated it. You can take your Xcode Playground, write several pieces of code and keep it on verified. Later you can export the whole code snippets, with the next command:

Reviewer: ```suggestion
Xcode Playgrounds are a great place where prototyping and trying new concepts. Oftentimes we want to share some Swift snippets; for this, `Carbon` is a cool tool and `nef` lovely integrated it. You can take your Xcode Playground, write several pieces of code and keep it verified. Later you can export all your code snippets, with the next command:
```

    <td>Font type</td>
    <td align="center">String</td>
    <td><code>firaCode</code> <code>hack</code> <code>inconsolata</code> <code>iosevka</code> <code>monoid</code> <code>anonymous</code> <code>sourceCodePro</code> <code>darkMono</code> <code>droidMono</code> <code>fantasqueMono</code> <code>ibmPlexMono</code> <code>spaceMono</code> <code>ubuntuMono</code></td>
    <td align="center"><code>firaCode</code></td>
  </tr>
  <tr>
    <td align="center"><code>--lines</code></td>
    <td>shows/hides number of lines in code snippet</td>
    <td align="center">Bool</td>
    <td><code>true</code> <code>false</code></td>
    <td align="center"><code>true</code></td>
  </tr>
  <tr>
    <td align="center"><code>--watermark</code></td>
    <td>shows/hides watermark in code snippet</td>
    <td align="center">Bool</td>
    <td><code>true</code> <code>false</code></td>
    <td align="center"><code>true</code></td>
  </tr>
</table>
  
#### Example of use

If we have a project created by `nef`, and a Xcode playground, with the next content:
```swift
let example = "This is an example"
print("nef is super cool: \(example)")

// nef is super cool: This is an example
```

> running the next command we will customize the `background color` to ![#d54048](https://placehold.it/15/d54048/000000?text=+) bow, `hides the number of lines` and set the export file to `size 3`:

Reviewer: ```suggestion
Running the following command we will customize the `background color` to ![#d54048](https://placehold.it/15/d54048/000000?text=+) bow, `hide the number of lines` and set the export file to `size 3`:
```
